.env.example

```diff
@@ -0,0 +1,16 @@
+# Full path to the LEGO Universe client
+CLIENT_PATH=/Users/someuser/LEGO Universe
+# Can improve build time
+BUILD_THREADS=1
+# Updates NET_VERSION in CMakeVariables.txt
+BUILD_VERSION=171022
+# make sure this is a long random string
+# grab a "SHA 256-bit Key" from here: https://keygen.io/
+ACCOUNT_MANAGER_SECRET=
+# Should be the externally facing IP of your server host
+EXTERNAL_IP=localhost
+# Database values
+MARIADB_USER=darkflame
+MARIADB_PASSWORD=SECRET_VALUE_CHANGE_ME
```

Comment:
Maybe there should be a comment that users should be careful with 'special' characters in passwords. For example when the password contains `@` (e.g. `test@1`), AccountManager fails to connect:

![AccountManager error screenshot](https://user-images.githubusercontent.com/11685886/147703933-a16abf2d-58da-4448-b5a1-96209f78a4d2.png)

Reply:
Yes